src/Core/MainLoop.js

```diff
@@ -176,6 +176,14 @@ MainLoop.prototype._step = function _step(view, timestamp) {
     this.needsRedraw = false;
     this.renderingState = RENDERING_PAUSED;
     const updateSources = new Set(view._changeSources);
+    let onlyHelpers = true;
+    for (const source of view._changeSources) {
```

Comment:
I don't understand the point of this ? Is it really needed ?

Reply:
I remove that, I added a simpler mechanism.